storage/namespace.go

```diff
@@ -216,18 +219,35 @@ func (n *dbNamespace) Bootstrap(
 	}
 	n.metrics.bootstrap.Success.Inc(1)
 
+	// Bootstrap shards using at least half the CPUs available
+	workers := pool.NewWorkerPool(int(math.Ceil(float64(runtime.NumCPU()) / 2)))
```

Comment:
I don't want it to round down to 0 if NumCPU() == 1.  e.g. https://play.golang.org/p/J8MuG1wGNF